.github/workflows/doctests.yml

```
@@ -4,12 +4,12 @@ on:
 
 jobs:
   run:
-    name: Doctests on ${{ matrix.os }} with Python ${{ matrix.python-version }}
+    name: OS ${{ matrix.os }} Python ${{ matrix.python-version }}
```

```suggestion
    name: ${{ matrix.os }} Python ${{ matrix.python-version }}
```

Since we're improving the display on the dashboard, I'd say 'OS' doesn't really add anything here; it's obvious from the name values (windows-latest, ubuntu-latest, etc) what the difference between worfklows is

---

Agree.

---

Done.